libvast/src/system/accountant.cpp

```diff
@@ -107,10 +109,10 @@ void accountant_state::command_line_heartbeat() {
   if (logger && logger->verbosity() >= CAF_LOG_LEVEL_INFO
       && accumulator.node.events > 0) {
     std::ostringstream oss;
-    auto node_rate = calc_rate(accumulator.node);
+    oss.imbue(std::locale(""));
+    auto node_rate = std::round(calc_rate(accumulator.node));
     oss << "ingested " << accumulator.node.events << " events"
-        << " at a rate of " << std::fixed << std::setprecision(2)
-        << node_rate / 1'000 << " events/ms";
+        << " at a rate of " << node_rate << " events/sec";
```

Comment:
There seems to be no reason to stream `<< " events" << " at a rate of "` instead of `<< " events at a a rate of "`.